src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -66,7 +66,7 @@ def create(self, username, password, ttl: Optional[int] = None):
             fsid = mgr.get('config')['fsid']
         except KeyError:
             fsid = ''
-        if max_attempt == 0 or mgr.ACCESS_CTRL_DB.get_attempt(username) < max_attempt:
+        if max_attempt == 0 or mgr.ACCESS_CTRL_DB.get_attempt(username) < max_attempt:  # pylint: disable=R1702,line-too-long # noqa: E501
```

Comment:
why is this?

Reply:
There were some linting errors in make check because of this - `too many nested blocks`